### PR TITLE
DEV-3181-new-go-exercise-printandmiss

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: solutions/printandmiss.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package solutions
+
+import "strings"
+
+func PrintAndMiss(arg string, loop int) string {
```
Comment (Hamzaelkhatri) [inline solutions/printandmiss.go:5]: Change name `loop` to, for example `, num`. It is confusing.

File: solutions/printandmiss.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	if arg == "" || loop < 0 {
+		return "Invalid Output\n"
+	}
+	if loop == 0 || loop > len(arg) {
+		return arg + "\n"
+	}
+	var _str strings.Builder
```
Comment (Hamzaelkhatri) [inline solutions/printandmiss.go:12]: you don't have to use `strings` library just use `str:=""`

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

File: solutions/printandmiss.go (hunk 1 above)
Comment (zainabdnaya) [reply solutions/printandmiss.go]: it's fixed

File: solutions/printandmiss.go (hunk 2 above)
Comment (zainabdnaya) [reply solutions/printandmiss.go]: it's fixed

**Review by Hamzaelkhatri — APPROVED** [review]

File: solutions/printandmiss.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package solutions
+
+func PrintAndMiss(arg string, num int) string {
+	if arg == "" || num < 0 {
```
Comment (jotapero) [inline solutions/printandmiss.go:4]: This condition needs to be fixed. Returns "Invalid Output" for negative numbers and it should return newline

**Review by jotapero — CHANGES_REQUESTED** [review]

File: solutions/printandmiss.go (hunk 3 above)
Comment (tiagocollot) [reply solutions/printandmiss.go:4]: It was an instructional error, as written in the subject file. It is now fixed.

**Review by jotapero — APPROVED** [review]
LGTM :alien:

**Review by eslopfer — CHANGES_REQUESTED** [review]
This review is pending until the subject is fixed.

**Pull request merged**